### ISSUE TITLE
Can only fetch a few option tickers

### ISSUE THREAD

**Issue by spaaaaam** [xemwebe/yahoo_finance_api#21]
Hello,

I am trying to fetch all tradeable option tickers on a given stock, but only manage to get a few of them:

The following code (from the repo examples):
```rs
    fn fetch_aapl_option_tickers(){
        let provider = yahoo::YahooConnector::new();
        let resp = tokio_test::block_on(provider.search_ticker("AAPL")).unwrap();
    
        println!("All tickers found while searching for 'AAPL':");
        for item in resp.quotes 
        {
            println!("{}", item.symbol)
        }
    }
```

returns this, the last 3 being put option tickers:
```cmd
All tickers found while searching for 'AAPL':
AAPL
AAPU
AAPD
AAPL.NE
AAPL221223P00130000
AAPL221223P00129000
AAPL221223P00132000
```

While I would like to fetch all available option tickers, visible [here](https://finance.yahoo.com/quote/AAPL/options?p=AAPL).

Is there a way to get them using a different request?

Thx a lot!

**Comment by xemwebe** [comment]
So far, the library only uses the (unofficial) JSON Rest API of yahoo! Finance, which seem to support only a small fractions of options. It would be possible to implement a html scraper to get the full option list directly from html sites like the one you have linked. Maybe a nice project for the next few days. I wil have look into it.

**Comment by spaaaaam** [comment]
That's very kind of you, thx!

**Comment by xemwebe** [comment]
I have just pushed a new version that offers an extra function to search all options. I have also added an example. Beware that this search is based on html scraper, which tend to be less stable than the REST API, but it should give you the same output as the web page. 

The last trading date is extracted as a string, not a date-time, since the chrono date does not seem to support parsing this specific format where the time zone is given as the name of the time zone. 

But within this limitation, it seems to work pretty well.
